### PR TITLE
Submitting my solution :)

### PR DESCRIPTION
**Pull request by Korak-997**
My solution :) will be so glad to hear any reviews :)

This is a PR template. If you are adding a solution link to the curriculum, leave this as is. If not, delete it and write the message you wish.

Thank you,

The Odin Project team.

### CODE REVIEW

**Comment by Korak-997** [comment]
TBH i do not understand what you guys actually mean am sorry :( could you please tell me how should i format it ?

File: web_development_101/the_front_end/project_html_css.md [hunk 1]
```diff
@@ -3246,7 +3246,7 @@ Submit a pull request with a link to your solution in this [file](https://github
 * [rkjenkins2's Solution (Easy)](https://github.com/rkjenkins2/google-homepage) - [View In Browser](https://rkjenkins2.github.io/google-homepage/)
 * [lalfiler's Solution (Easy)](https://github.com/lalfiler/google-homepage) - [View In Browser](https://lalfiler.github.io/google-homepage/)
 </details>
-
+https://github.com/Korak-997/google-website-test
```
Comment (I3uckwheat) [inline web_development_101/the_front_end/project_html_css.md:3249]: Notice how your link doesn't look like the other student solutions above it?
You've also put it outside the `</details>` tag.

Here's an example of a properly formatted submission:

`* [lalfiler's Solution (Easy)](https://github.com/lalfiler/google-homepage) - [View In Browser](https://lalfiler.github.io/google-homepage/)`

**Comment by Korak-997** [comment]
[lalfiler's Solution (Easy)](https://github.com/Korak-997/google-website-test) - [View In Browser](https://korak-997.github.io/google-website-test/)

**Comment by I3uckwheat** [comment]
@Korak-997 This should be done within the text file itself. You need to edit the file, and push to this branch. Commenting here is just a comment.

If we click "merge" we will still be merging in broken markdown

**Pull request merged**